css/pane_tab_content.css

```diff
@@ -25,7 +25,7 @@ body.styled-scrollbars .oblogger-content {
     padding: 0 4px 70px;
 }
 
-.oblogger-content .hidden { display:none; }
+.oblogger-content .hidden { display:none !important; }
```

Comment:
Was this the linch pin that was making it not work yesterday?

Reply:
it was! i could keep moving around the .hidden css whenever we use it (or write multiple of them) but labeling it important should just make it work regardless of its location.